src/main/flight/mixer_init.h

```diff
@@ -38,11 +38,15 @@ typedef struct mixerRuntime_s {
     float deadbandMotor3dHigh;
     float deadbandMotor3dLow;
 #ifdef USE_DYN_IDLE
-    float idleMaxIncrease;
+    float dynIdleMaxIncrease;
     float idleThrottleOffset;
-    float idleMinMotorRps;
-    float idleP;
-    float oldMinRps;
+    float dynIdleMinRps;
```

Comment:
Can you please rename these back? There is a lot going on in this pull request already, and having arbitrary renames added into the mix just makes it way harder to see what was actually changed.

Reply:
The new names aren't arbitrary.
They indicate that the parameter is part of the dynamic idle code, and better indicate what they actually do.
I changed them mostly because it makes the code easier to follow.

Reply:
Anyhow, the large-scale rename makes it hard to review this, so I think changing the names back for this pull request is a better approach. If the value of better names outweighs the cost of inconvenience to users when parameters that keep their function / scaling get renamed then this can always be done in a separate pull request later.

Reply:
I apologise for the large-scale renaming.
In most cases, the old names don't match the function of the parameter in the new code, and actually were incorrect in the old code.  
I agree that the two quoted above could be changed back, but is the translation of `idleMaxIncrease` into `dynIdleMaxIncrease `, and  `idleMinMotorRps ` into `dynIdleMinRps` so tough to follow?  
With all new parameters starting with `dynIdle...` or `dyn_idle...`I found the code easier to follow, to be honest.

Reply:
It just makes it hard to figure out what is actual functional change, and what is just a cosmetic rename.

Reply:
the code is functionally completely different, nothing like before, so both have to be sort of evaluated differently.
The old code had a very noisy D signal being added to an error signal, and both integrated; if the integral is positive, then the motors are boosted via a throttle lift.  In a sense the D becomes a P signal after integration, and the error becomes a classic I signal after integration. 
Due to the gating, and noise on D, the outcome wasn't easy to predict.  Spikes on the D element would push the integral briefly upwards, and give an earlier start, after a throttle chop, than one might otherwise get from a simple P I controller.  
The change I made is to use a classical PID controller with classical P, I and D elements.  In practice, D from rpm is very very noisy, and simple addition doesn't work well.  The peak of un-delayed D comes at the steepest part of the rpm fall, and since motors fall sort of exponentially, we get too much D where we don't really want it.  I learned that a simple PI controller alone does an exceptionally good job.  Due to the gating effect of not allowing the output below zero, we can smooth and delay D and add it effectively only at the start, as a kind of kick to pre-empt a fall below setpoint RPM.  In most tests, very little D is actually needed so long as I can be held close to the needed value.  But since 99% of the time, while flying, the RPM is way above this controller's setpoint, a continuously accumulating I would be massively high all the time.  Hence the constraints on I which mostly intend to keep I close to a useful value all the time.  
It wasn't easy for me to appreciate the subtlety of the original code, and very difficult to improve on it.  I found it much easier to understand once it was in familiar P, I and D values.